Makefile

```diff
@@ -28,9 +28,15 @@ build:
 	rm -rf ./dist
 	hugo -s site --verbose --destination ../dist/html --printUnusedTemplates --printPathWarnings --gc
 
+
+.PHONY: test
+test:
+	npm install
```

Comment:
The build error is because I have created a GitHub Action (@hanzei requested) to build and check the generated HTML files using HTMLProofer. 

I have tested the `test` command. The `npm install` won't cause any issues.